src/server/v2.0/handler/handler.go

```diff
@@ -31,6 +35,10 @@ func New() http.Handler {
 		log.Fatal(err)
 	}
 
+	api.RegisterMiddleware("CopyArtifact", middleware.Chain(quota.CopyArtifactMiddleware(), blob.CopyArtifactMiddleware()))
```

Comment:
is it possible to merge these two middlewares? They share some queries, I worried about the performance impact.

Reply:
This are two things, quota can be disabled, blob can not be disabled , gc depent on blobs.

Performance problem can be solved by other methods.